template.yaml

```diff
@@ -76,6 +76,24 @@ Resources:
               - method.request.querystring.language
               - method.request.querystring.title
 
+  NvaCristinOneProjectFunction:
```

Comment:
Curious to see what kind of OpenApi documentation will come up from this setting. We should test it and see if we need to change it in a subsequent PR